nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -351,13 +370,25 @@ def __init__(
             raise ValueError(f"conv_norm_type={norm_type} is not valid!")
 
         self.activation = Swish()
-        self.pointwise_conv2 = nn.Conv1d(
-            in_channels=dw_conv_input_dim, out_channels=d_model, kernel_size=1, stride=1, padding=0, bias=True
-        )
+        if self.conv_pointwise_type == 'conv1d':
+            self.pointwise_conv2 = nn.Conv1d(
+                in_channels=dw_conv_input_dim, out_channels=d_model, kernel_size=1, stride=1, padding=0, bias=True
+            )
+        elif self.conv_pointwise_type == 'linear':
+            self.pointwise_conv2 = torch.nn.Linear(dw_conv_input_dim, d_model)
+        else:
+            raise ValueError(
+                f"'{conv_pointwise_type}' is not not a valid value for 'conv_pointwise_type', "
+                f"valid values can be from ['conv1d', 'linear']"
+            )
 
     def forward(self, x, pad_mask=None, cache=None, cache_next=None):
-        x = x.transpose(1, 2)
-        x = self.pointwise_conv1(x)
+        if self.conv_pointwise_type == 'conv1d':
```

Comment:
There weren't transposes before for conv1d, why are they there now ?

Reply:
if conv_pointwise_type == 'conv1d', there is no change. If conv_pointwise_type == 'linear', then we transpose..